### PR TITLE
fix: test on windows

### PR DESCRIPTION
**Pull request by fengjiachun**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

As title said.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by WenyXu — APPROVED** [review]

**Review by waynexia — APPROVED** [review]

**Review by fengjiachun — COMMENTED** [review]
LGTM

File: src/common/datasource/src/object_store.rs [hunk 1]
```diff
@@ -44,17 +54,47 @@ pub fn parse_url(url: &str) -> Result<(String, Option<String>, String)> {
 }
 
 pub fn build_backend(url: &str, connection: &HashMap<String, String>) -> Result<ObjectStore> {
-    let (schema, host, _path) = parse_url(url)?;
+    let (schema, host, path) = parse_url(url)?;
+    let (root, _) = find_dir_and_filename(&path);
 
     match schema.to_uppercase().as_str() {
```
Comment (killme2008) [inline src/common/datasource/src/object_store.rs:60]: So we don't support azblob, aliyun oss etc. in v0.4? @waynexia @fengjiachun

Comment (killme2008) [reply src/common/datasource/src/object_store.rs:60]: Looks like it's only used in file table.

Comment (waynexia) [reply src/common/datasource/src/object_store.rs:60]: I'm not sure. But it won't be too much work to add support for other backends that opendal supported

**Pull request merged**